src/index.js

```diff
@@ -1,7 +1,15 @@
 "use strict";
+const fs = require('fs');
+const path = require('path');
+const { remote } = require('electron');
 
-if (process.env.DEVMODE) {
+const isDevMode = function() {
+  return remote.process.argv[2] == '--dev'
+};
+if (isDevMode()) {
   require("@babel/register");
+  const dotenv = require('dotenv');
+  dotenv.config();  
 }
```

Comment:
Ah, found `--dev` mode. Could you just say what's happening here?

Reply:
Thanks, I added a comment. This loads the `.env` file that has some env vars pointing to your local invest binaries.